pipert2/core/base/rpc_pipe_wrapper.py

```diff
@@ -0,0 +1,35 @@
+from zerorpc import Server
+from pipert2.core.base.pipe import Pipe
+from pipert2.utils.consts import START_EVENT_NAME, STOP_EVENT_NAME
+
+
+class RPCPipeWrapper:
```

Comment:
You can make the class inherit from the zerorpc server instead of holding it. [see here](https://github.com/0rpc/zerorpc-python/blob/master/tests/test_server.py#L42)
I think you should change it to that.

(Don't forget to update the demos if you decide to do that)